editor/components/TableColumnVisibilityMenu.tsx

```diff
@@ -200,7 +200,7 @@ export default function TableColumnVisibilityMenu({
     <Dropdown>
       <Dropdown.Toggle
         variant="outline-primary"
-        className="border-0 hide-toggle"
+        className="border-0 hide-toggle pb-2"
```

Comment:
I am seeing the slightest difference from this that seems negligible to me. Was this padding not looking right to you before?

Reply:
Yeah, the gear was not rendering center aligned and it was bothering me

without the padding:
<img width="720" height="288" alt="Screenshot from 2025-08-14 16-52-50" src="https://github.com/user-attachments/assets/edf88665-04be-489e-bef7-57cf94970d1f" />

with the padding:
<img width="720" height="288" alt="Screenshot from 2025-08-14 16-53-04" src="https://github.com/user-attachments/assets/1c74db1f-3bdb-4ad2-a5f6-31770f6b55c9" />

i tried using the d-flex align-items-center utility but that made the button move and look squatty

Reply:
I was curious where the random extra space was coming from, like why it was unbalanced without the `pb-2` 

<img width="97" height="110" alt="Screenshot 2025-08-15 at 1 35 22 PM" src="https://github.com/user-attachments/assets/bbfa6fe3-f306-48b1-8214-5ce149cbdebf" />

and i narrowed it down to (I think) that all `img, svg` get a `vertical-align: middle;` once I remove that in the developer tools its centered in the button without the pb-2. But dunno how to do that in css so this padding sounds good to me